pytensor/graph/utils.py

```diff
@@ -420,3 +431,92 @@ def toposort(prereqs_d):
             "some orderings contain invalid elements."
         )
     return seq
+
+
+def graph_replace(
+    outputs: Sequence["Variable"],
+    replace: Dict["Variable", "Variable"],
+    strict=True,
+    return_unused=False,
+) -> Union[Tuple[List["Variable"], Dict["Variable", "Variable"]], List["Variable"]]:
+    """Replace variables in ``outputs`` by ``replace`` in a single pass.
```

Comment:
```suggestion
    """Replace variables in ``outputs`` by ``replace``.
```

Is the `single pass` important to know about/ understand when choosing to use this function?